packages/typegpu/tests/tgsl/wgslGenerator.test.ts

```diff
@@ -81,12 +81,12 @@ describe('wgslGenerator', () => {
       floatLiteral: { value: '12.5', wgsl: '12.5', dataType: abstractFloat },
       scientificLiteral: {
         value: '12e10',
-        wgsl: '12e10',
-        dataType: abstractFloat,
+        wgsl: '120000000000',
+        dataType: abstractInt,
       },
       scientificNegativeExponentLiteral: {
         value: '12e-4',
-        wgsl: '12e-4',
+        wgsl: '0.0012',
```

Comment:
I am wondering why it chose decimal notation here, since the scientific notation is shorter 🤔. Any thoughts? @reczkok